examples/brute_force/brute_force.cpp

```diff
@@ -0,0 +1,166 @@
+/****************************************************************************
+ * Copyright (c) 2012-2019 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <ArborX_BruteForce.hpp>
+#include <ArborX_LinearBVH.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include <boost/program_options.hpp>
+
+#include <stdlib.h>
+#include <unistd.h>
+
+struct Dummy
+{
+  int count;
+};
+
+using MemorySpace = Kokkos::HostSpace;
+using ExecutionSpace = Kokkos::DefaultHostExecutionSpace;
+
+template <>
+struct ArborX::AccessTraits<Dummy, ArborX::PrimitivesTag>
+{
+  using memory_space = MemorySpace;
+  using size_type = typename MemorySpace::size_type;
+  static KOKKOS_FUNCTION size_type size(Dummy const &d) { return d.count; }
+  static KOKKOS_FUNCTION Point get(Dummy const &, size_type i)
+  {
+    return {{(float)i, (float)i, (float)i}};
+  }
+};
+
+template <>
+struct ArborX::AccessTraits<Dummy, ArborX::PredicatesTag>
+{
+  using memory_space = MemorySpace;
+  using size_type = typename MemorySpace::size_type;
+  static KOKKOS_FUNCTION size_type size(Dummy const &d) { return d.count; }
+  static KOKKOS_FUNCTION auto get(Dummy const &, size_type i)
+  {
+    return attach(
+        intersects(Sphere{{{(float)i, (float)i, (float)i}}, (float)i}), i);
```

Comment:
Is there are particular motivation to use spheres with radius `i` (instead of something fixed, maybe 1) here?